EcommerceStore.Application/Services/BrandService.cs

```diff
@@ -0,0 +1,82 @@
+using EcommerceStore.Application.Exceptions;
+using EcommerceStore.Application.Interfaces;
+using EcommerceStore.Application.Models.InputModels;
+using EcommerceStore.Application.Models.ViewModels;
+using EcommerceStore.Domain.Entities;
+using EcommerceStore.Domain.Interfaces;
+
+namespace EcommerceStore.Infrastucture.Services
+{
+    public class BrandService : IBrandService
+    {
+        private readonly IBrandRepository _brandRepository;
+
+        public BrandService(IBrandRepository brandRepository)
+        {
+            _brandRepository = brandRepository;
+        }
+
+        public async Task CreateBrandAsync(BrandInputModel brandIm)
+        {
```

Comment:
Please add validation for update/Create that there are no existing brands with this name